### ISSUE TITLE
VCL for CNN

### ISSUE THREAD

**Issue by kandeldeepak46** [nvcuong/variational-continual-learning#6]
Has anyone tried the CNN implementation of VCL?

**Comment by thangbui** [comment]
Yes. See this for example: https://github.com/TyXe-BDL/TyXe/blob/master/examples/vcl.py,  https://arxiv.org/pdf/2110.00276.pdf

This paper https://proceedings.mlr.press/v162/rudner22a/rudner22a.pdf also had some VCL numbers on CNNs + image datasets.
